.github/workflows/agenda.yaml

```diff
@@ -0,0 +1,53 @@
+name: Agenda Generator
+
+on: # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '15 12 * * 1' # run every Monday at 12:15 PM UTC
+  workflow_dispatch:
```

Comment:
I'd suggest to add a manual input here for the date. We could use this to manually schedule a Primaza Community Call that's not on a Monday.

Reply:
good idea, for example for this first one it will be manual

Reply:
What about now: https://github.com/dperaza4dustbit/primaza/discussions/30

Reply:
great